doc/source/data/loading-data.rst

```diff
@@ -375,7 +375,7 @@ Ray Data interoperates with libraries like pandas, NumPy, and Arrow.
             )
 
         You can also create a :class:`~ray.data.dataset.Dataset` from a list of regular
-        Python objects.
+        Python objects. In the schema, the default column name will be "item".
```

Comment:
Nit: Here and elsewhere:
```suggestion
        Python objects. In the schema, the column name defaults to "item".
```

We adhere to the Google technical writing style guide, which recommends active voice and present tense